src/config.rs

```diff
@@ -12,6 +12,7 @@ use crate::subscriber::NopEventFormatter;
 /// - Telemetry config
 /// - JSON formatter
 pub struct SubscriberConfig<T> {
+    pub country: Option<Country>,
```

Comment:
note that if we want to strictly follow semver this is a breaking change and would require a major bump

I don't think we need to care though

Reply:
Right, is there a way to make it non-breaking...?

Reply:
As I said I think it's fine to just say "screw semver", especially since this is an internal crate.

I don't think there is a way to not make this breaking other than not making the field public, or going back in time and adding a [#[non_exhaustive]](https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute) attribute on the struct